cmd/root.go

```diff
@@ -32,10 +48,37 @@ var rootCmd = &cobra.Command{
 		if verbose {
 			log.SetLevel(log.DebugLevel)
 		}
+		err = configureLogrusHooks()
+		if err != nil {
+			return errors.Wrap(err, "Unable to configure Logrus Hooks")
+		}
 		return nil
 	},
 }
 
+func configureLogrusHooks() error {
+	// Load Sentry Env
+	sentryEnv, err := loadSentryEnv()
+	if err != nil {
+		return err
+	}
+	// if env var not set, ignore
+	if sentryEnv.DSN == "" {
+		return nil
+	}
+
+	sentryHook, err := logrus_sentry.NewSentryHook(sentryEnv.DSN, []logrus.Level{
+		logrus.PanicLevel,
+		logrus.FatalLevel,
+		logrus.ErrorLevel,
+	})
+
+	if err == nil {
+		log.AddHook(sentryHook)
+	}
```

Comment:
```suggestion
        if err != nil {
           logrus.Errorf("Error configuring Sentry")
           return nil
        }
        log.AddHook(sentryHook)
```
I wonder if we should do something like that. That way we don't silently ignore when there is an error. What do you think?